.gitignore

```diff
@@ -2,3 +2,5 @@ wagtail_nhsuk_frontend.egg-info
 wagtailnhsukfrontend/static/wagtailnhsukfrontend/css/wagtail-nhsuk-frontend.min.css
 testapp/db.sqlite3
 testapp/media/images
+.vscode
```

Comment:
@AdamChrimes  These ignores are local to your system, not the project.
You can set up your own local gitignore: http://egorsmirnov.me/2015/05/04/global-gitignore-file.html